docs/cnfc.md

```diff
@@ -0,0 +1,35 @@
+# CNFC
+
+Keycloak is currently a CNCF incubating project. This document provides useful information and links around CNFC.
+
+## Useful links
+
+* [CLO Monitor](https://clomonitor.io/projects/cncf/keycloak)
+* [CNCF Artwork](https://github.com/cncf/artwork/tree/master/projects/keycloak)
+* [CNCF Landscape](https://landscape.cncf.io/?selected=keycloak)
+* [CNCF Online Programs Guidelines](https://github.com/cncf/foundation/blob/main/online-programs-guidelines.md)
+* [CNFC Service Desk](https://cncfservicedesk.atlassian.net/servicedesk/customer/portals)
+* [DevStats](https://keycloak.devstats.cncf.io/)
+* [Slack](https://cloud-native.slack.com/) (current channels #keycloak, #keycloak-dev, and #keycloak-maintainers)
+* [Services for CNCF Projects](https://www.cncf.io/services-for-projects/)
+* [Schedule time with the CNCF Projects Team](http://project-meetings.cncf.io/)
+
+## Maintainers
+
+Maintainers should be listed in the following places:
+
+* [Keycloak Maintainers List](https://github.com/keycloak/keycloak/blob/main/MAINTAINERS.md)
+* [CNFC Maintainers List](https://maintainers.cncf.io/)
```

Comment:
Should we be linking that despite Keycloak not being present at https://maintainers.cncf.io/?

Reply:
@vmuzikar I was able to find Keycloak here: https://github.com/cncf/foundation/blob/main/project-maintainers.csv#L1333

Reply:
Sorry, I see it now too. I don't know why Ctrl-F does not work properly there. 🙈